src/src/PluginStructs/P109_data_struct.h

```diff
@@ -50,24 +49,29 @@ const char flameimg[] PROGMEM = {
 # define P109_CONFIG_DISPLAYTYPE        PCONFIG(2)
 # define P109_CONFIG_CONTRAST           PCONFIG(3)
 # define P109_CONFIG_RELAYPIN           PCONFIG(4)
-# define P109_CONFIG_SETPOINT_DELAY     PCONFIG(5)
+# define P109_CONFIG_TIMEOUT            PCONFIG(5)
+# define P109_CONFIG_MAX_TIMEOUT        PCONFIG(6)
+# define P109_CONFIG_STEP_TIMEOUT       PCONFIG(7)
 # define P109_CONFIG_HYSTERESIS         PCONFIG_FLOAT(0)
+# define P109_CONFIG_SETPOINT_DELAY     PCONFIG(8)
+# define P109_MODE_STATE_INITIAL        PCONFIG(9)
```

Comment:
`PCONFIG()` only handles 8 values, 0..7, so using above 7 will cause issues with the next task values. You can use `PCONFIG_LONG()` in range 0..4, if not already used for something else.